### DISCUSSION TITLE
What will be the description that the commitments will carry?

### DISCUSSION THREAD

**Discussion by w2d-ar** [Web-Wise-Dynamics/officialweb#11] [Q&A] [answered]
Please describe which texts and their respective descriptions will be taken to take into account in the **Code Review**

**Comment by thecoderbutcher** [comment, marked as answer]
Here ready the messages that we define for the commits:

- **feat**: For all new functionalities
- **fix**: For arranged bugs that have been informed by the PM or detected after raising a functionality
- **hotfix**: A utmost error given that the BUG reaches production and must proceed immediately with the Code Review and its subsequent Merge

Each message will be structured as follows

> Prefix: Message

where the prefix will be one of the detailed above and the message describing and in detail the action carried out

**Reply by w2d-ar** [reply]
Excellent!
